microcloud/test/includes/microcloud.sh

```diff
@@ -2,7 +2,7 @@
 unset_interactive_vars() {
   unset LOOKUP_IFACE LIMIT_SUBNET SKIP_SERVICE EXPECT_PEERS \
     SETUP_ZFS ZFS_FILTER ZFS_WIPE \
-    SETUP_CEPH CEPH_WARNING CEPH_FILTER CEPH_WIPE \
+    SETUP_CEPH CEPH_WARNING CEPH_FILTER CEPH_WIPE SETUP_CEPHFS \
```

Comment:
In this commit, is the cephfs pool actually checked for valid functionality, i.e by creating a cephfs volume and attaching it to a running VM and container? I'd like to avoid the issues we saw prior to the 1.1 release where we found LXD VMs couldnt be launched on ceph. Apologies if I am not reading this correctly and its already there.

Reply:
Added now.